chef-config/lib/chef-config/dist.rb

```diff
@@ -22,5 +22,15 @@ class Dist
 
     # Enable forcing Chef EULA
     ENFORCE_LICENSE = true
+
+    # The client's alias (chef-server)
+    SERVER = "chef-server".freeze
+
+    # The configuration directory for chef-server
+    SERVER_CONF_DIR = "chef-server".freeze
```

Comment:
im actually not sure here but the idea was this would go here: https://github.com/chef/chef-server/blob/66216dedeb96ebcbc9ca25a2b920f23598d1453d/oc-chef-pedant/spec/running_configs/manage_spec.rb#L5 for /etc/`opscode`/